config/examples/stilts_example.conf

```diff
@@ -0,0 +1,38 @@
+// This config is for STILTS training for BERT -> MNLI -> RTE
```

Comment:
Mention that we're using base. Most users will want to mess with this then switch to large.